resources/sass/scrollbar.scss

```
@@ -0,0 +1,21 @@
+/* width */
+::-webkit-scrollbar {
+    width: 10px;
+  }
+  
+  /* Track */
+  ::-webkit-scrollbar-track {
+    background: #f1f1f100; 
+  }
+   
+  /* Handle */
+  ::-webkit-scrollbar-thumb {
+    background: $light-accent-1; 
+  }
+  
+  /* Handle on hover */
+  ::-webkit-scrollbar-thumb:hover {
+    background: lightgray; 
+  }
```

```suggestion
/* Track */
::-webkit-scrollbar-track {
  background: #f1f1f100; 
}
   
/* Handle */
::-webkit-scrollbar-thumb {
  background: $light-accent-1; 
}
  
/* Handle on hover */
::-webkit-scrollbar-thumb:hover {
  background: lightgray; 
}
```
Can we remove these extra spacing?